EventSourcing.Cosmos.Tests/CosmosRecordAttributeTests.cs

```diff
@@ -0,0 +1,30 @@
+using EventSourcing.Core;
+using EventSourcing.Core.Tests;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace EventSourcing.Cosmos.Tests;
+
+public class CosmosRecordAttributeTests: RecordAttributeTests
```

Comment:
Downside of having these different tests classes in the core file is that the Cosmos configuration and initialisation needs to be done multiple times. It would be nicer to have one tests suite class that a particular implementation has to implement. Kind of like the ```EventStoreTests```, where I used a partial class to split one class up in multiple files.